polkadot/xcm/src/v3/junction.rs

```diff
@@ -243,6 +293,19 @@ impl TryFrom<OldBodyPart> for BodyPart {
 	}
 }
 
+impl From<NewBodyPart> for BodyPart {
```

Comment:
Again, if they're the same, just re-export the type from the new version module.